app/controllers/user/UserStats.controller.ts

```diff
@@ -33,6 +79,21 @@ export async function create(request: IUserRequest, response: Response) {
     return response.json(stats);
 }
 
+/**
+ * @api {get} /users/stats/coins Show the number of coins the user has.
+ * @apiName GetCoinsOfUser
+ * @apiGroup User
+ * 
+ * @apiParam {string} user The id of the user. // Not sure about this param. Might be all users since there's no path param?
```

Comment:
Hi @VerumEmanio,

Avoid adding additional bloat the docs, if you have queries use the issue.
Its based on the authenticated user and not all users. So no Id.
Additionally the response is not json, its just a string based response.